src/components/MaximWPproj2/styles/Container.styled.tsx

```diff
@@ -0,0 +1,168 @@
+import { Button } from 'antd'
+import styled from 'styled-components'
+
+export const Hrstyle = styled.hr`
+  width: 250px !important;
+  border-top: 1px solid;
+  color: rgb(219, 219, 219);
+`
+export const Hrstyle2 = styled.hr`
+  width: 250px !important;
+  border-top: 1px solid;
+  color: rgb(168, 173, 254);
```

Comment:
if the only difference between the 2 styles are the color, consider passing it as a prop so theres less duplicated code

eg 
```
export const StyledHr = styled.hr<{color: 'color1' | 'color2'}>`
//your styles
`
```